### PR TITLE
Users can see a list of events

### PR DESCRIPTION
**Pull request by dvidgry**
-added a view for the list of events
-moved the login/logout buttons into the layout

### CODE REVIEW

File: app/views/events/index.html.erb [hunk 1]
```diff
@@ -2,10 +2,15 @@
...
 
-<% if current_user %>
-  <%= form_with url: '/users/sign_out', method: "delete", local: true do |form| %>
-    <%= form.submit value: "Log out" %>
-  <% end %>
-<% else %>
- <%= link_to 'Log in', new_user_session_path %>
+<% @events.reverse.each do |event| %>
+  <%= event.title %>
+  <br>
```
Comment (axcochrane) [inline app/views/events/index.html.erb:7]: when we write CSS we can abstract the <br> tags into a line_height attribute

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -11,6 +11,13 @@
   </head>
 
   <body>
+    <% if current_user %>
+      <%= form_with url: '/users/sign_out', method: "delete", local: true do |form| %>
```
Comment (axcochrane) [inline app/views/layouts/application.html.erb:15]: local: true -- what does this do, I should probably know :S

**Review by enonnai — CHANGES_REQUESTED** [review]
As now the 'Log In' code has been moved to application.html.erb, it is displayed across the app. Is that intended?

**Comment by BenNoonan1991** [comment]
Looks really good guys, one thing though is that at the moment it looks like you can add an event without being logged in, it might be good to add an error message if you click 'Add an event' when you're not logged in.

**Review by axcochrane — APPROVED** [review]
:shipit:

File: app/views/events/index.html.erb [hunk 3]
```diff
@@ -2,10 +2,15 @@
...
 <br>
 
-<% if current_user %>
-  <%= form_with url: '/users/sign_out', method: "delete", local: true do |form| %>
-    <%= form.submit value: "Log out" %>
-  <% end %>
-<% else %>
- <%= link_to 'Log in', new_user_session_path %>
+<% @events.reverse.each do |event| %>
+  <%= event.title %>
```
Comment (enonnai) [inline app/views/events/index.html.erb:6]: Shall we have 'Event Title:' etc displaying before the name of event, and for the other fields too?

**Comment by axcochrane** [comment]
@BenNoonan1991 -- very good point, we have a ticket on the trello board which @jonathanelliot is taking care of next! 👍

**Review by enonnai — APPROVED** [review]
:shipit:

**Pull request merged**